README.md

```diff
@@ -27,11 +27,17 @@ _Also available in Objective-C with [VTAcknowledgementsViewController](https://g
 
 ## How to Install
 
-CocoaPods is the most common solution to install this library.
+### Swift Package Manager
```

Comment:
Thank you for updating the README!

Reply:
A la fin du README, j'ai oublié, il faudrait remplacé "requires iOS 8.0" par "requires iOS 9.0".